harness/features/allFeatures.local.test.ts

```diff
@@ -31,13 +31,7 @@ describe('allFeatures Tests - Local', () => {
         let testClient: LocalTestClient
 
         beforeEach(async () => {
-            testClient = new LocalTestClient(sdkName)
-            scope
```

Comment:
why is this not setting up the config mock anymore?

Reply:
it's set in the client constructor